.github/workflows/build.yml

```diff
@@ -76,24 +76,24 @@ jobs:
           - worker: docker+containerd  # same as docker, but with containerd snapshotter
             pkg: ./tests
             mode: experimental
-          - worker: "docker@27.5"
+          - worker: "docker@28.5"
             pkg: ./tests
-          - worker: "docker+containerd@27.5"  # same as docker, but with containerd snapshotter
+          - worker: "docker+containerd@28.5"  # same as docker, but with containerd snapshotter
             pkg: ./tests
-          - worker: "docker@27.5"
+          - worker: "docker@28.5"
             pkg: ./tests
             mode: experimental
-          - worker: "docker+containerd@27.5"  # same as docker, but with containerd snapshotter
+          - worker: "docker+containerd@28.5"  # same as docker, but with containerd snapshotter
             pkg: ./tests
             mode: experimental
-          - worker: "docker@26.1"
+          - worker: "docker@27.5"
             pkg: ./tests
-          - worker: "docker+containerd@26.1"  # same as docker, but with containerd snapshotter
+          - worker: "docker+containerd@27.5"  # same as docker, but with containerd snapshotter
```

Comment:
Wondering if we should include a 25.0 somewhere; 26.x and 27.x are EOL, but 25.0 is maintained by mirantis and AWS